README.md

```diff
@@ -81,6 +81,8 @@ Our goals for [v0.39 release cycle](https://github.com/microsoft/PowerToys/issue
 
 #### Highlights from v0.39 / v0.41
 
+The PowerToys team delayed our 0.39 release. We learned a lot, but items took longer than expected. We had more we wanted done before the next release, so this longer time allowed us to get in more amazing pull requests by you, the community, to add / improve functionality for 0.41.
```

Comment:
i'd adjust the wording here.  We decided that we wanted to do more for 0.39 so merged it into our 0.41 release. .....